packages/migrate/migrations/svelte-5/migrate.js

```diff
@@ -0,0 +1,122 @@
+import fs from 'node:fs';
+import { Project, ts, Node } from 'ts-morph';
+import { log_on_ts_modification, update_pkg } from '../../utils.js';
+
+export function update_pkg_json() {
+	fs.writeFileSync(
+		'package.json',
+		update_pkg_json_content(fs.readFileSync('package.json', 'utf8'))
+	);
+}
+
+/**
+ * @param {string} content
+ */
+export function update_pkg_json_content(content) {
+	return update_pkg(content, [
```

Comment:
this removed the newline from the end of the file when I ran it on `svelte-awesome`. I think it's nicer to have a newline at the end of each file